timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -53,6 +53,13 @@ def predict(self, X: pd.DataFrame) -> np.ndarray:
         assert isinstance(X, pd.DataFrame)
         return self.predictor.predict(X).values
 
+    @classmethod
+    def load(cls, path: str) -> "TabularEstimator":
+        # There is no need to pass init & fit kwargs since predictor is already trained
```

Comment:
This will require either writing a custom serialization logic in `AbstractMLForecastModel.save`, or trying to recreate the dicts based on the properties of the loaded `TabularPredictor` object. I feel like both options introduce a lot of complexity, so I would rather just ignore it since the `predictor_init_kwargs` / `predictor_fit_kwargs` are not used anywhere.

Reply:
I don't understand why custom `save` introduces complexity? We have the `save_pkl` and `load_pkl` helpers?